x-pack/plugins/gis/public/_main.scss

```diff
@@ -8,6 +8,10 @@
   overflow: hidden;
 }
 
+.fullScreen {
```

Comment:
Danger Will Robinson... This class is way too generic. Please add your `gis` prefix.

Reply:
Just remember, that all CSS for all plugins gets loaded everywhere.